### DISCUSSION TITLE
How to set a circular polarized laser in WarpX?

### DISCUSSION THREAD

**Discussion by jiecaix** [BLAST-WarpX/warpx#4610] [Q&A] [answered]
I want to set a circularly polarized laser, but the input file don't have a circular polarization option in laser module. Could I set 2 laser with a CEP difference of `pi` between the 2 laser?

**Comment by aodhanmci** [comment, marked as answer]
you can set 2 lasers with equal amplitude, orthogonal polarisations and phi0 of pi/2
